src/io/flutter/run/daemon/FlutterAppManager.java

```diff
@@ -41,7 +41,7 @@
 
   private static final Gson GSON = new Gson();
 
-  private final Logger myLogger = Logger.getInstance("#io.flutter.run.daemon.FlutterAppManager");
+  private static final Logger LOG = Logger.getInstance(FlutterAppManager.class.getName());
```

Comment:
If I'm understanding this right, we're doing away with unique loggers per instance.  I *think* that's OK but it may be confusing in the cases where we're debugging multiple apps.

(Or wait, is there an app manager per app, or?)

EDIT: probably OK either way.  Feel free to ignore!